### PR TITLE
added link to Flannel on Windows guide and clarification on ongoing DNS work

### PR DESCRIPTION
**Pull request by daschott**
1. Added Flannel How-to guide for deploying Kubernetes on Windows
2. Added minor clarification to ongoing DNS work

### CODE REVIEW

**Comment by stewart-yu** [comment]
The contents are  LGTM

**Comment by daschott** [comment]
assign @chenopis

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ There are several supported network configurations with Kubernetes v1.9 on Windo
 The selection of which network configuration and topology to deploy depends on the physical network topology and a user's ability to configure routes, performance concerns with encapsulation, and requirement to integrate with third-party network plugins.
 
 ### Future CNI Plugins
-An additional two CNI plugins [win-l2bridge (host-gateway) and win-overlay (vxlan)] are in [PR review](https://github.com/containernetworking/plugins/pull/85). These two CNI plugins, when ready, can either be used directly or with Flannel.
+An additional two CNI plugins [win-l2bridge (host-gateway) and win-overlay (vxlan)] are in [PR review](https://github.com/containernetworking/plugins/pull/85). These two CNI plugins, when ready, can either be used directly or with Flannel. In the meantime, binaries with the PR's have been published under the [Microsoft/SDN](https://github.com/Microsoft/SDN/tree/master/Kubernetes/flannel) repository. Step-by-step instructions of how to use the win-l2bridge plugin using Flannel can be found in the [Host-Gateway](#host-gateway-topology) section below.
```
[the remark this answers is not in the document]
Comment (chenopis) [reply content/en/docs/getting-started-guides/windows/_index.md]: +1 What's the estimated time for when these two plugins will be ready?

[the remark this answers is not in the document]
Comment (daschott) [reply content/en/docs/getting-started-guides/windows/_index.md]: The links are already there today, it is not something new I am adding! Are you suggesting I should remove them?

[the remark this answers is not in the document]
Comment (chenopis) [reply content/en/docs/getting-started-guides/windows/_index.md]: In general, it is not good to have time dependent content in the documentation. In the very near future when the two CNI plugins are ready, this paragraph will need to be updated. I suggest rewriting this as:

```
### CNI Plugins

Two CNI plugins, win-l2bridge (host-gateway) and win-overlay (vxlan),
can be used directly or with Flannel. Their binaries are published under the
[Microsoft/SDN](https://github.com/Microsoft/SDN/tree/master/Kubernetes/flannel)
repository. Step-by-step instructions on how to use the win-l2bridge
plugin with Flannel can be found in the [Host-Gateway](#host-gateway-topology)
section below.
```

**Comment by zacharysarah** [comment]
@daschott 👋 Please feel free to reopen this PR when you're ready to incorporate review feedback.

**Pull request closed without merging**